kernel/aster-nix/src/syscall/mod.rs

```diff
@@ -231,10 +359,22 @@ impl SyscallArgument {
     }
 }
 
-pub fn handle_syscall(context: &mut UserContext) {
+/// The information of current process, posix thread, thread and task.
+pub struct CurrentInfo<'a, 'b, 'c, 'd> {
```

Comment:
Using a single lifetime should be enough. These fields are constructed at the same time and compiler should choose the shortest lifetime.

Further, how do you like the name `CurrentRefs`? This struct is indeed store references to some structs of current.